### ISSUE TITLE
Custom statement with label will not continue post init

### ISSUE THREAD

**Issue by morbil** [renpy/renpy#4032]
So, I have a statement meant for creating some read-only data, but also starts off some actual script after it. so it defines a label, and then during the game can jump to the statement to continue after to execute the script portions. 

At least, that's the idea. In execution, although it runs and sets up the data as expected, when it jump/calls the label, it will execute the block and then end the game. I have tried defining `next` explicitly as a pass as well as giving it a different label and it simply does not execute. Only execute fires when it's jumped to before ending the game (and returning to the menu). 

Here is a simple example to demonstrate:

```renpy
python early:
    class FooCommand:
        @staticmethod
        def parse(l):
            label = l.label_name(True)
            l.set_global_label(label)
            return label

        @staticmethod
        def execute(po):
            print("Exec!")

        @staticmethod
        def execute_init(po):
            print("Init!")

        @staticmethod
        def label(po):
            return po

    renpy.register_statement(
        "foo",
        parse=FooCommand.parse,
        execute=FooCommand.execute,
        label=FooCommand.label,
        execute_init=FooCommand.execute_init,
        init=True,
        init_priority=-100,
    )

foo the_label

"Made it to next!"
return

label start:
    "About to call"
    call the_label
    "Made it back!"
return
```
If you run this you get "Exec!" and "Init!" on init as expected, and then once you get to the `call the_label` you get the "Exec!" again and then you are back in the main menu. If you remove the `execute_init`, `init`, and `init_priority` arguments from the `register_statement` call, it works as expected but without triggering on init

Of course, this is only when you jump/call the label. If you simply do:
```renpy
label start:
    "About to exec"
    foo the_label
    "Made it to next!"
    "Made it back!"
return
```
In this case, it will "Exec!" once it hits that statement, but continue as normal to the next statement. 

Am I missing something specific here? Like I said, I tried dealing with `next` explicitly, but the effect was the same. 

Note: This is Renpy 7.5.3

**Comment by renpytom** [comment]
You can remove the init=True and init_priority=-100 from your statement. 

The former because init=True is equal to: 

```renpy
init:
     foo the_label
```

Which takes the statement out of the normal flow of control. If you remove init=True, things should work, as the statement will then be followed by the return. 

As for `init_priority` - it doesn't actually work with execute_init. I just fixed that in 0f8576f18, but until that lands execute_init will run at init 0. 

I'm about to land another commit that documents this better.

**Comment by morbil** [comment]
Thank you for the clarification, as well as the init priority fix as I probably would have run into that right afterward. As an aside, is there a way to generate an CDS that acts like `default`, that is that can mark itself to be added to `default_statements`? `execute_init` is more or less enough to act like `define`, but I don't see how to get one that acts like `default` and get called at the point of `execute_default_statement`
